apiExamples/icon.html

```diff
@@ -0,0 +1,12 @@
+<auro-button aria-label="wifi">
+  Activate WiFi
+  <auro-icon customColor category="in-flight" name="wifi" slot="icon"></auro-icon>
```

Comment:
```
  <auro-icon customColor category="in-flight" name="wifi" slot="icon"></auro-icon>
```

What is `slot="icon"` for?

Reply:
Answered my own question. That is some old API stuff there. An icon as content works both in the main slot and the icon-specific slot. Ok then.